.github/workflows/create-codespaces-prebuild.yml

```diff
@@ -0,0 +1,17 @@
+name: Create Codespaces Prebuild
```

Comment:
Whoa - is this what it looks like? Exciting!

We might later add more to the build (eg libs.tests) if we're typically picking up a cached image.

Reply:
If the prebuild fails, how will we know?

Reply:
> is this what it looks like?

yes.

> If the prebuild fails, how will we know?

It is a GitHub action. I would hope we can set up rules when it fails to notify someone.

Reply:
https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#github-actions-notification-options